ghost/tpl/values.toml

```diff
@@ -1,3 +1,6 @@
+ghostHostname = ""
```

Comment:
Since the Ghost image param is named `GHOST_HOST`, this should probably be named `ghostHost`